app/views/contacts/index.html.erb

```diff
@@ -12,8 +12,8 @@
         <% @users.each do |user| %>
             <!--for each user -->
             <tr>
-                <td><%= user.first_name %></td>
-                <td><%= user.last_name %></td>
+                <td><%= user.first_name.capitalize %></td>
+                <td><%= user.last_name.capitalize %></td>
```

Comment:
At least the last name capitalization has to be in an `if` and `else`. It's possible that someone doesn't have a value set when they are a pending user, just a first name. But we could do it for both just to be safe. Or you could override the `.first_name` function in the `user.rb` model.